### ISSUE TITLE
Ad CORS headers for Javascript API

### ISSUE THREAD

**Issue by emersion** [nokonoko/Pomf#25]
Add CORS headers to allow JS scripts to post to upload endpoint

**Comment by tdaede** [comment]
I would like this, preferably on Uguu too.

**Comment by nokonoko** [comment]
@avail mind helping on this one?

**Comment by avail** [comment]
CORS has to be added through the server's config (nginx/apache), can't do shit about that.

**Comment by tdaede** [comment]
No, it's just HTTP headers. You can generate the headers with PHP, if no content has been emitted yet.

**Comment by avail** [comment]
@tdaede good point, didn't think about that.
